Upstream's patch, can be removed with next update.

See:
https://github.com/LibreCAD/LibreCAD/commit/6c392e903e162b9283e88f53006e929663f2e883
and
https://www.mail-archive.com/bug-guix@gnu.org/msg09231.html

--- librecad/src/ui/forms/qg_commandwidget.cpp.orig	2020-06-21 11:32:28.000000000 -0400
+++ librecad/src/ui/forms/qg_commandwidget.cpp	2020-06-21 11:33:08.000000000 -0400
@@ -24,6 +24,8 @@
 **
 **********************************************************************/
 #include "qg_commandwidget.h"
+
+#include <QAction>
 #include <QKeyEvent>
 #include <algorithm>

--- librecad/src/ui/generic/widgetcreator.cpp.orig	2020-06-21 11:34:38.000000000 -0400
+++ librecad/src/ui/generic/widgetcreator.cpp	2020-06-21 11:34:54.000000000 -0400
@@ -27,6 +27,8 @@
 #include "widgetcreator.h"
 #include "ui_widgetcreator.h"

+#include <QAction>
+#include <QActionGroup>
 #include <QSettings>
 #include <QLineEdit>
 #include <QPushButton>
